.husky/commit-msg

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
again, quick walk-through please

Reply:
basically, this checks the default commit message (other comment) and checks that the commit message matches the correct format. If it is a merge, this is skipped to avoid problems

Reply:
Fallback?

Reply:
meaning?

Reply:
i mean what happens if the default commit message created in `post-checkout` is empty?

Reply:
also, im seeing a different error message than the one you wrote?
![image](https://user-images.githubusercontent.com/1917227/149946887-b7f7f2ff-977e-484e-afb8-16afeb44b517.png)

Reply:
- [ ] check why the error message would talk about no matching files rather than the wrong commit message (tried committing something without "#123" upfront

Reply:
i pushed some changes but couldnt get this error. online search suggests running npm i again